.vscode/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
The .vscode folder contains items that are used when developing the azure function in VSCode